contrib/mkimage-yum.sh

```diff
@@ -113,22 +113,8 @@ rm -rf "$target"/sbin/sln
 rm -rf "$target"/etc/ld.so.cache "$target"/var/cache/ldconfig
 mkdir -p --mode=0755 "$target"/var/cache/ldconfig
 
-version=
```

Comment:
Perhaps making "version" an option to be passed to the script instead? And if not set, try to detect it?